### ISSUE TITLE
run_all_postannotation- pho variable is blank

### ISSUE THREAD

**Issue by itang1** [SeedlingsBabylab/datavyu_scripts#18]
**the problem:**
with the recent new introduction of the pho variable, `run_all_postannotation.rb` is angry that all of their entries are blank (see below). this clutter distracts from any real actual problems that it might point out.

**possible solutions:**
a) tell `run_all_postannotation.rb` to forgive the blanks
b) computationally go change all the pho entries in all the files to NA

```
*************************
Running Script: run_all_postannotation.rb on project: 35_09_sparse_code
*************************

check_codes (Found empty code): [Column]: labeled_object       [Variable]: pho    [Cell#]: 1
check_codes (Found empty code): [Column]: labeled_object       [Variable]: pho    [Cell#]: 2
check_codes (Found empty code): [Column]: labeled_object       [Variable]: pho    [Cell#]: 3
check_codes (Found empty code): [Column]: labeled_object       [Variable]: pho    [Cell#]: 4
check_codes (Found empty code): [Column]: labeled_object       [Variable]: pho    [Cell#]: 5
check_codes (Found empty code): [Column]: labeled_object       [Variable]: pho    [Cell#]: 6
check_codes (Found empty code): [Column]: labeled_object       [Variable]: pho    [Cell#]: 7
check_codes (Found empty code): [Column]: labeled_object       [Variable]: pho    [Cell#]: 8
check_codes (Found empty code): [Column]: labeled_object       [Variable]: pho    [Cell#]: 9
check_codes (Found empty code): [Column]: labeled_object       [Variable]: pho    [Cell#]: 10
.
.
.
this goes on for every cell
```

**Comment by ebergelson** [comment]
@sarpu let's have it only  print this for CHI cells

**Comment by itang1** [comment]
yay excellent, those `found empty code`s now only appear for untranscribed CHI's, as intended.

(though in case it is of interest, i will point out that in 08_15, and possibly other files, their transcriptions appear in its corresponding point cell (such as with cells 320-321). i am not sure why some of the pho transcriptions got migrated over from the corresponding point cell while others did not.)

<img width="241" alt="Screen Shot 2020-04-17 at 6 05 21 PM" src="https://user-images.githubusercontent.com/24839492/79617833-68ede800-80d6-11ea-9b9e-7cbdcb53b774.png">


anyways, now there's just some complaints saying `code cannot contain space`. this complaint is directed at CHI cells that already have the %pho transcribed (such as with cell 7)

(disregard the `not a valid id code` errors, that is addressed in #20)
<img width="250" alt="0815cell7" src="https://user-images.githubusercontent.com/24839492/79616799-e9f7b000-80d3-11ea-89fe-b4ac26296cd3.png">




```
*************************
Running Script: run_all_postannotation.rb on project: 08_15_sparse_code
*************************

Cell# 2:  "0x19b304" is not a valid id code (this id is not in the list of used IDs)
check_codes: code cannot contain space: [Column]: labeled_object       [Variable]: pho    [Cell#]: 7
check_codes: code cannot contain space: [Column]: labeled_object       [Variable]: pho    [Cell#]: 9
check_codes (Found empty code): [Column]: labeled_object       [Variable]: pho    [Cell#]: 11
Cell# 12:  "0xb49408" is not a valid id code (this id is not in the list of used IDs)
check_codes: code cannot contain space: [Column]: labeled_object       [Variable]: pho    [Cell#]: 22
check_codes: code cannot contain space: [Column]: labeled_object       [Variable]: pho    [Cell#]: 24
check_codes: code cannot contain space: [Column]: labeled_object       [Variable]: pho    [Cell#]: 27
check_codes: code cannot contain space: [Column]: labeled_object       [Variable]: pho    [Cell#]: 30
check_codes: code cannot contain space: [Column]: labeled_object       [Variable]: pho    [Cell#]: 52
check_codes: code cannot contain space: [Column]: labeled_object       [Variable]: pho    [Cell#]: 57
check_codes: code cannot contain space: [Column]: labeled_object       [Variable]: pho    [Cell#]: 60
Cell# 62:  "0x24ae43" is not a valid id code (this id is not in the list of used IDs)
check_codes: code cannot contain space: [Column]: labeled_object       [Variable]: pho    [Cell#]: 91
check_codes: code cannot contain space: [Column]: labeled_object       [Variable]: pho    [Cell#]: 103
check_codes: code cannot contain space: [Column]: labeled_object       [Variable]: pho    [Cell#]: 105
check_codes: code cannot contain space: [Column]: labeled_object       [Variable]: pho    [Cell#]: 109
check_codes: code cannot contain space: [Column]: labeled_object       [Variable]: pho    [Cell#]: 124
check_codes: code cannot contain space: [Column]: labeled_object       [Variable]: pho    [Cell#]: 126
check_codes: code cannot contain space: [Column]: labeled_object       [Variable]: pho    [Cell#]: 146
check_codes: code cannot contain space: [Column]: labeled_object       [Variable]: pho    [Cell#]: 149
check_codes: code cannot contain space: [Column]: labeled_object       [Variable]: pho    [Cell#]: 152
check_codes: code cannot contain space: [Column]: labeled_object       [Variable]: pho    [Cell#]: 203
check_codes: code cannot contain space: [Column]: labeled_object       [Variable]: pho    [Cell#]: 217
check_codes (Found empty code): [Column]: labeled_object       [Variable]: pho    [Cell#]: 218
check_codes: code cannot contain space: [Column]: labeled_object       [Variable]: pho    [Cell#]: 229
check_codes: code cannot contain space: [Column]: labeled_object       [Variable]: pho    [Cell#]: 238
Cell# 245:  "0x221f2f" is not a valid id code (this id is not in the list of used IDs)
check_codes: code cannot contain space: [Column]: labeled_object       [Variable]: pho    [Cell#]: 251
check_codes: code cannot contain space: [Column]: labeled_object       [Variable]: pho    [Cell#]: 274
check_codes: code cannot contain space: [Column]: labeled_object       [Variable]: pho    [Cell#]: 282
check_codes: code cannot contain space: [Column]: labeled_object       [Variable]: pho    [Cell#]: 286
check_codes: code cannot contain space: [Column]: labeled_object       [Variable]: pho    [Cell#]: 289
check_codes: code cannot contain space: [Column]: labeled_object       [Variable]: pho    [Cell#]: 311
check_codes: code cannot contain space: [Column]: labeled_object       [Variable]: pho    [Cell#]: 313
check_codes (Found empty code): [Column]: labeled_object       [Variable]: pho    [Cell#]: 314
check_codes: code cannot contain space: [Column]: labeled_object       [Variable]: pho    [Cell#]: 316
check_codes (Found empty code): [Column]: labeled_object       [Variable]: pho    [Cell#]: 317
Cell# 318:  "0xfac8b2" is not a valid id code (this id is not in the list of used IDs)
check_codes: code cannot contain space: [Column]: labeled_object       [Variable]: pho    [Cell#]: 319
check_codes (Found empty code): [Column]: labeled_object       [Variable]: pho    [Cell#]: 320
Cell# 321:  "0x82a177" is not a valid id code (this id is not in the list of used IDs)
check_codes: code cannot contain space: [Column]: labeled_object       [Variable]: pho    [Cell#]: 322
check_codes (Found empty code): [Column]: labeled_object       [Variable]: pho    [Cell#]: 323
Cell# 324:  "0xeaa2c3" is not a valid id code (this id is not in the list of used IDs)
check_codes: code cannot contain space: [Column]: labeled_object       [Variable]: pho    [Cell#]: 325
check_codes (Found empty code): [Column]: labeled_object       [Variable]: pho    [Cell#]: 326
Cell# 327:  "0x1bcc9a" is not a valid id code (this id is not in the list of used IDs)
check_codes: code cannot contain space: [Column]: labeled_object       [Variable]: pho    [Cell#]: 328
check_codes (Found empty code): [Column]: labeled_object       [Variable]: pho    [Cell#]: 329
check_codes: code cannot contain space: [Column]: labeled_object       [Variable]: pho    [Cell#]: 331
check_codes: code cannot contain space: [Column]: labeled_object       [Variable]: pho    [Cell#]: 337


Script has finished running.
```

**Comment by sarpu** [comment]
The bug with some pho transcriptions still appearing in their own cell was addressed in add_pho repo.
